openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -2293,6 +2268,178 @@ The following is a non-normative example of a Credential Response containing a C
+    claim. Valid values defined by this specification are `string`, `number`,
+    and image media types such as `image/jpeg` as defined in IANA media type
+    registry for images
+    (https://www.iana.org/assignments/media-types/media-types.xhtml#image).
+    Other values MAY also be used.
+  * `display`: OPTIONAL. Array of objects, where each object contains display
+    properties of a certain claim in the Credential for a certain language.
+    Below is a non-exhaustive list of valid parameters that MAY be included:
+     * `name`: OPTIONAL. String value of a display name for the claim.
+     * `locale`: OPTIONAL. String value that identifies language of this object
+       represented as language tag values defined in BCP47 [@!RFC5646]. There
+       MUST be only one object for each language identifier.
+
+The rules defined in (##claims-description-processing) apply.
+
+## Processing Rules for Claims Description Objects {#claims-description-processing}
+
+The order of claims description objects in the `claims`
+array is used by the Wallet to determine the order in which the claims are
+displayed to the End-User, unless another mechanism is defined by the profile.
+
+When a repeated or contradictory claim description is provided, the processing
+MUST be aborted. This is in particular the case if 
+
+ - the same claim is addressed by two or more claims description objects in the
+   `claims` array, or
+ - there is a claims description object with a `path` that addresses a set of
+   claims in an array (using `null`, as defined in (#claims_path_pointer)) and
+   another object that uses a non-negative integer to address a specific claim
+   in the same array, or
+ - there is a claims description object indicating that a certain claim is an array
+   (implied by using `null` or a non-negative integer in the `path`), and another
+   object indicating that the same claim is an object (implied by using a string
+   in the `path`).
+
+
+# Claims Path Pointer {#claims_path_pointer}
+
+A claims path pointer is a pointer into the JSON structure of the Verifiable
+Credential, identifying one or more claims. A claims path pointer MUST be a
+non-empty array of strings and non-negative integers. A string value 
+indicates that the respective key is to be selected, a null value
+indicates that all elements of the currently selected array(s) are to be selected;
+and a non-negative integer indicates that the respective index in an array is to be selected. The path
+is formed as follows:
+
+Start with an empty array and repeat the following until the full path is formed. 
+
+  - To address a particular claim within an object, append the key (claim name)
+    to the array.
+  - To address an element within an array, append the index to the array (as a
+    non-negative, 0-based integer).
+  - To address all elements within an array, append a null value to the array.
+
+Verifiers MUST NOT point to the same claim more than once in a single query.
```

Comment:
i think this should be removed or rephrased as Verifiers are not relevant to creating claims descriptions

Reply:
verifiers need to be changed to Credential issuers.